### ISSUE TITLE
Documentation link error 404

### ISSUE THREAD

**Issue by HendrikHuebner** [teamseodo/vue3-tree#122]
Looks like the link in the README leads nowhere.

**Comment by alpercun** [comment]
> Looks like the link in the README leads nowhere.

You're right. We forgot to update the new page in the readme section. We will fix this as soon as possible. Until it is fixed, you can access the new link [here](https://wopehq.github.io/vue3-tree/).

**Comment by HendrikHuebner** [comment]
Thanks. It would also be very helpful if you added some documentation about how to style the tree view. Like what classes are there for example

**Comment by alpercun** [comment]
> Thanks. It would also be very helpful if you added some documentation about how to style the tree view. Like what classes are there for example

When the document will be updated, we can update it by taking this suggestion into consideration. Thank you for your suggestion. 💫
